microcosm_flask/swagger/parameters/__init__.py

```diff
@@ -1,6 +1,6 @@
 from collections.abc import Mapping
 from functools import lru_cache
-from pkg_resources import iter_entry_points
+from importlib.metadata import entry_points
```

Comment:
now using `importlib` instead of `pkg_resources`, this was last remaining references to this now deprecated package.